app/assets/javascripts/admin/addon/templates/components/ip-lookup.hbs

```diff
@@ -20,12 +20,18 @@
 
         <dt>{{i18n "ip_lookup.location"}}</dt>
         <dd>
-          {{#if this.location.location}}
-            <a href="https://maps.google.com/maps?q={{this.location.latitude}},{{this.location.longitude}}" rel="noopener noreferrer" target="_blank">
-              {{this.location.location}}
+          {{#if this.location.no_license}}
+            <a href="https://meta.discourse.org/t/configure-maxmind-for-reverse-ip-lookups/173941" rel="noopener noreferrer" target="_blank">
+              {{i18n "this.location.no_license"}}
             </a>
           {{else}}
-            {{i18n "ip_lookup.location_not_found"}}
+            {{#if this.location.location}}
```

Comment:
```hbs
{{else if this.location.location}}
```
is supported from handlebars v3 on, until then this nested solution is required: https://stackoverflow.com/questions/10736907/handlebars-js-else-if

Reply:
That is interesting. We have `{{else if...` in our codebase, so I assume it works.

Reply:
Ah my mistake, handlebars v4.x is already used across Discourse. I'll update the PR accordingly.